### PR TITLE
Update about us page

### LINKED ISSUE #5: Update About Us page with team details
- Add your own details. Include a suitable photo as described here.
- Including the the supervisor/lecturer is optional.
- The filename of the profile photo should be docs/images/githbub_username_in_lower_case.png. 
- Note the need for lower case e.g. JohnDoe123 -> docs/images/johndoe123.png not docs/images/JohnDoe123.png.
- If your photo is in jpg format, name the file as .png anyway.
- Indicate the different roles played and responsibilities held by each team member.
- You can reassign these roles and responsibilities (as explained in Admin  Project Scope) later in the project, if necessary.

### PR DESCRIPTION
**Pull request by Eriksen2411**

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -7,53 +7,45 @@ We are a team based in the [School of Computing, National University of Singapor
 
 You can reach us at the email `seer[at]comp.nus.edu.sg`
 
-## Project team
+## CoLAB Project team
 
-### John Doe
+### Fang Samuel
```
Comment (samuelfangjw) [inline docs/AboutUs.md]: I think putting my full name here will be better! (consistent with the other names)

`Fang Junwei, Samuel`

File: docs/AboutUs.md [hunk 2]
```diff
@@ -7,53 +7,45 @@ We are a team based in the [School of Computing, National University of Singapor
...
-
-### Jane Doe
-
-<img src="images/johndoe.png" width="200px">
-
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
-
-* Role: Team Lead
+* Role: Deliverables and deadlines
```
Comment (samuelfangjw) [inline docs/AboutUs.md]: perhaps we can change the phrasing of the roles for all of us to `In charge of ...`. e.g. In charge of documentation, In charge of Integration. Currently the role sections sound a bit weird for all of us.

File: docs/AboutUs.md [hunk 3]
```diff
@@ -7,53 +7,45 @@ We are a team based in the [School of Computing, National University of Singapor
...
-<img src="images/johndoe.png" width="200px">
+<img src="images/lirc572.png" width="200px">
 
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](https://github.com/lirc572)]
+[[portfolio](team/lirc572.md)]
+
+* Role: Code quality
+* Responsibilities: Test
```
Comment (samuelfangjw) [inline docs/AboutUs.md]: Perhaps Tests instead of test would be better here.

**Review by samuelfangjw — COMMENTED** [review]
Great work! Just a few minor comments :) 

I think for the responsibilities portion we can be a bit more detailed, but let's leave updating that portion to until we have finished deciding what features we are going to work on.

File: docs/AboutUs.md (hunk 2 above)
Comment (Eriksen2411) [reply docs/AboutUs.md]: yeah i think its weird yesterday too, but not sure thanks for the comment

File: docs/AboutUs.md (hunk 1 above)
Comment (vevek) [reply docs/AboutUs.md]: Agreed. Full name for everyone would be great.

File: docs/AboutUs.md (hunk 3 above)
Comment (vevek) [reply docs/AboutUs.md]: Agreed

**Review by vevek — CHANGES_REQUESTED** [review]
Great work :) . Can I also suggest adding the role "Developer" to everyone. I feel that it would help reflect our roles clearly.

File: docs/AboutUs.md (hunk 1 above)
Comment (Eriksen2411) [reply docs/AboutUs.md]: @vevek Am i addressing your full name correctly

Comment (vevek) [reply docs/AboutUs.md]: @Eriksen2411 Yes :)

**Comment by Eriksen2411** [comment]
> Great work :) . Can I also suggest adding the role "Developer" to everyone. I feel that it would help reflect our roles clearly.

Agreed

File: docs/AboutUs.md (hunk 2 above)
Comment (vevek) [reply docs/AboutUs.md]: In that case would it be 

In charge of: Deliverables and deadlines
vs
Role: Deliverables and deadlines
vs
Role: In charge of Deliverables and Deadlines

**Review by vevek — CHANGES_REQUESTED** [review]
Looks great. I think the In should be lower case. Instead of "In charge of" change to "in charge of".

Also, I think it would look better if its like Deliverables and Deadlines, compared to deliverables and deadlines in lower case.

**Comment by vevek** [comment]
LGTM.

**Comment by Eriksen2411** [comment]
closes #5

**Pull request merged**